components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -2514,3 +2540,323 @@ def object_table_query(request, objtype, objid, conn=None, **kwargs):
     tableData['parentId'] = ann['parentId']
     tableData['addedOn'] = ann['addedOn']
     return tableData
+
+
+def build_url(request, name, api_version, **kwargs):
```

Comment:
Not denying its utility. `api.github.com` has the luxury of always being available under that URL prefix and strict control therein however. We do not. If you can find a way to do this without relying on `build_absolute_uri()`, great. Otherwise my position is that we compromise and go with the absolute path portion of the URL.